src/main/java/Buddy.java

```diff
@@ -0,0 +1,123 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
+// camelCase used as a coding standard
+
+public class Buddy {
+    public static void main(String[] args) {
+        String greeting = "Hello there! I'm Buddy\n"
+                + "How may I assist you?";
+        String exitMessage = "Hope I was of help to you! Have a great day and see you again, Buddy :)";
+        String divider = "________________________________________________________________________________";
+
+        System.out.println(divider);
+        System.out.println(greeting);
+        System.out.println(divider);
+
+        Task[] listOfThings = new Task[100];  // why cannot private static?
+        int currentPosition = 0;            // why cannot private static?
+        String command;
+        Scanner in = new Scanner(System.in);
+        command = in.nextLine();
+
+        while (! command.equals("bye")){
+            int index = 1;
+            if (command.equals("list")){
+                for (int i = 0; i < currentPosition; i++){ // while not null
```

Comment:
Perhaps a space could be added before the opening braces, like so: `else {`?
I believe there are multiple occurrences below as well.